doc/radosgw/cloud-transition.rst

```diff
@@ -0,0 +1,347 @@
+================
+Cloud Transition
+================
+
+This feature enables data transition to a remote cloud service as part of `Lifecycle Configuration <https://docs.aws.amazon.com/AmazonS3/latest/dev/object-lifecycle-mgmt.html>`__ via `Storage Classes <https://docs.ceph.com/en/latest/radosgw/placement/#storage-classes>`__. The transition is unidirectional; data cannot be transitioned back from the remote zone. The goal of this feature is to enable data transition to multiple cloud providers. The currently supported cloud providers are those that are compatible with AWS (S3).
```

Comment:
we should use relative links instead of `https://docs.ceph.com/en/latest/radosgw`, because this will eventually become the 'pacific' or 'quincy' doc, and we don't want the link to take you to a newer release

there's already a `_storage_classes` label in placement.rst, and s3.rst links to it like this:
```
| **Storage Class**               | Supported       | See :ref:`storage_classes`             |
```
that renders as `See Storage Classes`. you can do the same here